.github/actions/publish-tag-and-release/action.yml

```diff
@@ -0,0 +1,92 @@
+#################################################################################
+#  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+#  Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+
+---
+name: "Publish Github release and tag"
+description: "Publish Github release and tag"
+inputs:
+  version:
+    required: true
+    description: "The version to be used in the tag and release publication"
+  token:
+    required: true
+    description: "Github token"
+  isLatest:
```

Comment:
nit: in other composite actions we used different case. the most used looks to be the snake_case. 
(for the record, in yaml files for some reasons I like kebab-case more, but snake will fit better in this project)

Reply:
[3ea619b](https://github.com/eclipse-tractusx/tractusx-edc/pull/1543/commits/3ea619b7c4e94c36c9efbb977336e6dbd521b639)